4.1/test/run

```diff
@@ -70,6 +94,7 @@ run "create_container test_welcome_page"
 CONTAINER_ARGS=
 for i in `seq 10`; do
     sleep 5
+curl $(get_container_ip 'test_welcome_page'):8080
```

Comment:
@hhorak, this is debuging output, right?

Maybe it could be done using ` |  tee output` instead of ` > output` on the following line.